docker-compose.yml

```diff
@@ -96,5 +98,35 @@ services:
       - "1025:1025"
       - "8025:8025"
 
+  keycloak:
```

Comment:
I guess that you don't really need this here, right?
Anyway, we have this file outdated and [we plan to remove it soon](https://issues.redhat.com/browse/THREESCALE-3246).